node/proxy/api/src/zrx.ts

```diff
@@ -0,0 +1,80 @@
+import axios, { isAxiosError } from 'axios'
+import { Axios } from 'axios'
+import { Request, Response } from 'express'
+
+const ZRX_API_KEY = process.env.ZRX_API_KEY
+
+if (!ZRX_API_KEY) throw new Error('ZRX_API_KEY env var not set')
+
+export class Zrx {
+  private axiosInstance: Axios
+
+  constructor() {
+    this.axiosInstance = axios.create({
+      headers: {
+        '0x-api-key': ZRX_API_KEY,
+      },
+    })
+  }
+
+  async handler(req: Request, res: Response): Promise<void> {
+    const parsedUrl = new URL('https://dummy.com'.concat(req.url))
```

Comment:
Is there any rationale here re: not using req.url directly?

Reply:
`req.url` doesn't include the base url and will not be parsed correctly by `URL`, so just added a dummy base url so we can use it and make sure there is no funny business with pulling off the query params (versus just split on `?`)